docs/admin/integrations/aci_setup.md

```diff
@@ -0,0 +1,137 @@
+
+```shell
+pip install nautobot-ssot[aci]
+```
+
+## Configuration
+
+Integration behavior can be controlled with the following settings:
+
+| Setting Name<br>(* required) | Type | Description |
+|---|:---:|---|
+| <p>aci_apics_*</p> |  | Per-APIC settings. See per-APIC settings section for details. |
+| aci_tag* | _String_ | Tag which is created and applied to all <br>synchronized objects. |
+| aci_tag_color* | _String_ | Hex color code used for the tag. |
+| aci_tag_up* | _String_ | Tag indicating the state applied to synchronized <br>interfaces. |
+| aci_tag_up_color* | _String_ | Tag color applied to the "UP" tag on interface <br>status. |
+| aci_tag_down* | _String_ | Tag indicating the state applied to synchronized <br>interfaces. |
+| aci_tag_down_color* | _String_ | Tag color applied to the "DOWN" tag on interface <br>status. |
+| aci_manufacturer_name* | _String_ | Manufacturer name. Specifically existing, or a new <br>one with this name will be created. |
+| aci_ignore_tenants* | _List[String]_ | List of ACI Tenants that should not be synchronized<br>from APIC. |
+| aci_comments* | _String_ | Comment added to synchronized objects. |
+
+Below is an example snippet from `nautobot_config.py` that demonstrates how to enable and configure Cisco ACI integration:
+
+```python
+PLUGINS_CONFIG = {
+    "nautobot_ssot": {
+        "enable_aci": True,
+        # URL and credentials should be configured as environment variables on the host system
+        "aci_apics": {x: os.environ[x] for x in os.environ if "APIC" in x},
+        # Tag which will be created and applied to all synchronized objects.
+        "aci_tag": "ACI",
+        "aci_tag_color": "0047AB",
+        # Tags indicating state applied to synchronized interfaces.
+        "aci_tag_up": "UP",
+        "aci_tag_up_color": "008000",
+        "aci_tag_down": "DOWN",
+        "aci_tag_down_color": "FF3333",
+        # Manufacturer name. Specify existing, or a new one with this name will be created.
+        "aci_manufacturer_name": "Cisco",
+        # Exclude any tenants you would not like to bring over from ACI.
+        "aci_ignore_tenants": ["common", "mgmt", "infra"],
+        # The below value will appear in the Comments field on objects created in Nautobot
+        "aci_comments": "Created by ACI SSoT Plugin",
+    }
+```
+
+### Per-APIC settings
+
+The APIC URL and credentials need to be created as environment variables on the host system.
+
+You can configure multiple APIC instances for synchronization. To do this, append `_` character, followed by an identifier, to the names of environment variables.
+
+In the example below, configured APIC uses `NTC` for an identifier. Instead of `NTC` you could, for example, use `CHCG01` to configure an APIC instance in your Chicago facility.
+
+```bash
+export NAUTOBOT_APIC_BASE_URI_NTC=https://aci.cloud.networktocode.com
```

Comment:
you could show the other example with devnet that is easier to reproduce

Reply:
Copied from the original plugin doc. Could you pleas provide more specific suggestion or ask someone with the devnet account?

Reply:
ok, if you copied the original one, let's keep it as it is